docs/deployment.md

```diff
@@ -87,7 +87,15 @@ Enter repository root directory
 
     $ cd <daac>-deploy
 
-Install packages with npm
+Packages are installed with npm. To view packages on npm, visit:
```

Comment:
Ok super nitpicky, but maybe just link then to the package list on npm. https://www.npmjs.com/search?q=%40cumulus

Otherwise good with me!